init.sql

```diff
@@ -0,0 +1,28 @@
+CREATE ROLE librarian WITH LOGIN SUPERUSER PASSWORD 'librarian';
+ALTER DATABASE library OWNER TO librarian;
+
+CREATE TABLE person (
+    person_id   SERIAL      PRIMARY KEY,
```

Comment:
Little remark (DRY), when you'll be write a SQL query for the table, you get such record:
`person.person_id`, it will work but to much characters for the programmer )